src/main/java/org/betonquest/betonquest/compatibility/mythicmobs/MythicMobsIntegrator.java

```diff
@@ -17,6 +19,9 @@ public void hook() {
         plugin.registerConditions("mythicmobdistance", MythicMobDistanceCondition.class);
         plugin.registerObjectives("mmobkill", MythicMobKillObjective.class);
         plugin.registerEvents("mspawnmob", MythicSpawnMobEvent.class);
+        if (Compatibility.getHooked().contains("ProtocolLib")) {
+            MythicHider.start();
+        }
```

Comment:
I think you need to start the hider in two scenarios, like the NPCHider is started twice. That is needed because we can not guarantee the order plugins are enabled.

And i think i also miss a reload of this?